wgpu-types/src/lib.rs

```diff
@@ -111,7 +111,7 @@ impl From<Backend> for BackendBit {
 
 /// Options for requesting adapter.
 #[repr(C)]
-#[derive(Clone, Debug, Default, PartialEq, Eq, Hash)]
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
```

Comment:
Unfortunately because of [this issue](https://github.com/rust-lang/rust/issues/26925), I have to manually implement `Default`.